docs/workflow.md

```diff
@@ -0,0 +1,372 @@
+- **CHT with LTFU configuration** - [https://interop-cht-test.dev.medicmobile.org/](https://interop-cht-test.dev.medicmobile.org/).
+
+## Steps
+
+The following steps assume that you successfully logged in into OpenHIM and the CHT instances.
+
+1.  Create an **Organization**
+
+    1. HTTP Request - First, make a request to create an **Endpoint** Resource in the mediator using Postman. You can view the API documentation for creating an endpoint [here](#endpoint-resource). Once you send the request, the mediator will return a JSON response containing the `id` of the newly created endpoint.
+
+    1. HTTP Request - Next, create an **Organization** Resource in the mediator using the `endpoint.id` returned from the previous request in the `endpoint.reference` field, replacing `${ENDPOINT_ID}` with the actual `id` of the endpoint you created in the previous step. Once you send the request, the mediator will return a JSON response containing the `id` of the newly created organization. You can view the API documentation for creating an organization [here](#organisation-resource).
+
+    1. It is important to note that you only need to create an **Organization** once, which you can use for future requests. So, after creating the **Organization**, you can save the `organization.identifier[0].value` value and use it for all future `ServiceRequest` requests.
+
+1.  Create a **Patient**
+
+    1. CHT - Log in to the CHT platform using the credentials for the `chw` user.
+    1. CHT - Navigate to the `People` tab in the CHT dashboard. From there, select a Facility where you want to create a new `Person`. Click on the `New Person` button and fill in the required details for the Person. Make sure to select `Patient` as the `Person`'s role for this flow.
+    1. CHT - Once you have created the new `Person`, you need to retrieve their unique identifier from the browser's URL. You can do this by copying the alphanumeric string that appears after `person/` in the URL. Keep this identifier safe as you will need it for the next steps.
+    1. OpenHIM Admin Console - To verify that the `Patient` creation was successful, navigate to the `Transaction Log` in the OpenHIM Admin Console. You should see two successful API calls recorded in the log, one to `/mediator/patient/` and one to `/fhir/Patient/`.
+       ![](./images/instance-patient.png)
+
+1.  Request the LTFU for the Patient
+
+    1. HTTP Request - To trigger the LTFU process for the newly created patient, you need to create a `ServiceRequest`. You can refer to the API documentation available [here](#service-request-resource) to learn how to create a `ServiceRequest`. Once the `ServiceRequest` is received by the mediator, it will initiate the LTFU workflow for the patient, which includes reminders for follow-up appointments and check-ins. Replace the `requester.reference` and the `subject.reference` with the `Organization` and `Patient` identifiers respectively.
```

Comment:
```suggestion
    1. HTTP Request - To trigger the LTFU process for the newly created patient, you need to create a `ServiceRequest`. You can refer to the API documentation available [here](#servicerequest-resource) to learn how to create a `ServiceRequest`. Replace the `requester.reference` and the `subject.reference` with the `Organization` and `Patient` identifiers respectively. Once the `ServiceRequest` is received by the mediator, it will initiate the LTFU workflow for the patient, which includes reminders for follow-up appointments and check-ins. 
```